patches/buildroot/0002-add-TOOLS_DIR-and-SYSROOT_DIR-to-control-output.patch

```diff
@@ -1,6 +1,6 @@
-From c137f14fade2ae3c23ab389b2e3e99f8bb5a55ab Mon Sep 17 00:00:00 2001
-From: Ben Cressey <bcressey@amazon.com>
-Date: Sun, 6 Aug 2017 23:35:06 +0000
+From 042b3f15465d1d93e0b99e0d659c7cb55eef3b15 Mon Sep 17 00:00:00 2001
+From: Martin Harriman <larvacea@mac.com>
```

Comment:
Please keep the author line intact; you can add notes below my sign-off and then add your own sign-off to capture your contribution.

E.g.
```
Signed-off-by: Ben Cressey <bcressey@amazon.com>
[larvacea:
  - rebase to x.y.z
  - fixed context
  - reworked blah
]
Signed-off-by: Martin Harriman <larvacea@mac.com>
```